gsb_research_hub_subtheme.react_behaviors.yml

```diff
@@ -69,7 +69,17 @@ banner_variant_info:  # part of the key after the :
         rh-banner-wrapper--background-cool-grey: Cool Grey
         rh-banner-wrapper--background-mist: Mist
         rh-banner-wrapper--background-driftwood: Driftwood
+        rh-banner-wrapper--background-driftwood-75: Driftwood 75%
+        rh-banner-wrapper--background-driftwood-50: Driftwood 50%
+        rh-banner-wrapper--background-driftwood-25: Driftwood 25%
+        rh-banner-wrapper--background-stone: Stone
```

Comment:
I'm happy that you are using the variant configuration!

Reply:
I think our most clever hack is figuring out that we can use states to hide/show elements.